packages/admin/admin/src/FinalForm.tsx

```diff
@@ -256,7 +259,7 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
             .then(
                 (data) => {
                     // for final-form undefined means success, an obj means error
-                    form.reset(values);
+                    form.reset(props.initialValues);
```

Comment:
Why do you need this change?

Reply:
I think this is resolved due to your changes.